### PR TITLE
zh: sync update diagnostic-tools/proxy-cmd doc to latest

### PR DESCRIPTION
**Pull request by shuaijinchao**
Please provide a description for what this PR is for.
sync update document content/zh/docs/ops/diagnostic-tools/proxy-cmd/index.md to latest

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by shuaijinchao** [comment]
@kebe7jun please review

File: content/zh/docs/ops/diagnostic-tools/proxy-cmd/index.md [hunk 1]
```diff
@@ -390,10 +390,10 @@ $ istioctl proxy-config bootstrap -n istio-system istio-ingressgateway-7d6874b48
 1. 使用`curl`测试 Istiod 的连接。下面的示例使用默认 Istiod 配置参数和启用相互 TLS 调用 v1 注册 API：
 
     {{< text bash >}}
-    $ kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n foo -- curl -sS istiod.istio-system:15014/debug/endpointz
+    $ kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n foo -- curl -sS istiod.istio-system:15014/version
     {{< /text >}}
 
-您应该收到一个响应，其中列出了网格中每个服务的“服务”和“端点”。
+你应该会收到列出 Istiod 版本的响应。
```
Comment (kebe7jun) [inline content/zh/docs/ops/diagnostic-tools/proxy-cmd/index.md:396]: use 您 instead of 你 in chinese docs.

Comment (shuaijinchao) [reply content/zh/docs/ops/diagnostic-tools/proxy-cmd/index.md:396]: should honorifics be avoided as much as possible in documentation?

Comment (shuaijinchao) [reply content/zh/docs/ops/diagnostic-tools/proxy-cmd/index.md:396]: thx~, #10952 has done this update, I will close this PR.

**Pull request closed without merging**